Source/SPFileManagerAdditions.m

```diff
@@ -171,7 +171,6 @@ - (NSStringEncoding)detectEncodingforFileAtPath:(NSString *)aPath
 	}
 	
 	NSData *startData = [detectorFileHandle readDataOfLength:500];
-	[detectorFileHandle release];
```

Comment:
Ahhh I think this was probably me. Had to rewrite the file handler to remove a deprecated and unsupported encoding-detection framework. Good catch!